tests/upgrade/integration/v2/loader.go

```diff
@@ -7,9 +7,10 @@ import (
 	integrationhelpers "github.com/cheqd/cheqd-node/tests/integration/helpers"
 	"github.com/cheqd/cheqd-node/x/did/client/cli"
 	didtypesv2 "github.com/cheqd/cheqd-node/x/did/types"
-	didtypesv1 "github.com/cheqd/cheqd-node/x/did/types/v1"
+
+	// didtypesv1 "github.com/cheqd/cheqd-node/x/did/types/v1"
 	resourcetypesv2 "github.com/cheqd/cheqd-node/x/resource/types"
-	resourcetypesv1 "github.com/cheqd/cheqd-node/x/resource/types/v1"
+	// resourcetypesv1 "github.com/cheqd/cheqd-node/x/resource/types/v1"
```

Comment:
Marking to remove commented out lines, effective for the whole file.

Reply:
Done